### ISSUE TITLE
Incorrect / misleading documentation about CP_HOME_DIR

### ISSUE THREAD

**Issue by wujek-srujek** [CocoaPods/CocoaPods#12339]
* [*] I've read and understood the [*CONTRIBUTING* guidelines and have done my best effort to follow](https://github.com/CocoaPods/CocoaPods/blob/master/CONTRIBUTING.md).

# Report

## What did you do?

I read the documentation FAQ section.

Here: https://guides.cocoapods.org/using/faq.html, under 'Can I change the default CocoaPods repositories folder' it says that using CP_HOME_DIR changes both repo and caches directory, but this is not correct - both by observing the behavior, and also evidenced by the comment here: https://github.com/CocoaPods/CocoaPods/issues/9225#issuecomment-539542493

Please update the documentation to be correct.

## What did you expect to happen?

I expected to get correct information.

## What happened instead?

I got incorrect / misleading information.

## CocoaPods Environment

Any.

## Project that demonstrates the issue

No project necessary.

**Comment by Vyazovoy** [comment]
I'm not sure that the described issue does exist.
Here is a simple experiment that you can run in your project:
```
rm -rf Pods
mkdir .cocoapods
CP_HOME_DIR=.cocoapods bundle exec pod install
```
Here is what I see in my directory tree:
<img width="145" alt="Screenshot 2024-04-11 at 11 14 51" src="https://github.com/CocoaPods/CocoaPods/assets/2086081/e716bf81-b2f7-4d5d-a5da-64d661fdae4d">

So, looks like the environment variable does what it supposed to.

**Comment by wujek-srujek** [comment]
I apologize, you are right.

In my case, as I was testing it the first time, the pods were already available in the project in the `Pods/` subfolder, so removing `~/Library/Caches/CocoaPods` and running `pod install` didn't create the caches dir. This got me searching and then I found the comment in one of the issues, which reassured me in my fallacy.

Now I tested again, after cleaning everything everywhere, and you are right, `CP_HOME_DIR` does contain both the repos and the caches.
